guides/common/modules/ref_customizing-the-registration-templates.adoc

```diff
@@ -21,9 +21,12 @@ You can configure the following global parameters by navigating to *Configure* >
 
 ifndef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `false`.
+Set the parameter type to `boolean` to override the parameter value.
 endif::[]
 ifdef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `true`.
+Set the parameter type to `boolean` to override the parameter value.
+If the parameter is set to `false`, it prevents hosts from appearing in the {Team} Inventory reports and also the Insights client from uploading reports to the Inventory of your {RHCloud}.
```

Comment:
This line applies to all builds.

Reply:
And I think this is the case where you want to use `Red Hat` instead of `{Team}` :)

Reply:
Indeed :smiley: